source/guide_tiddlywiki5.rst

```diff
@@ -0,0 +1,187 @@
+.. highlight:: console
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+
+.. tag:: wiki
+.. tag:: web
+.. tag:: lang-javascript
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/tiddlywiki.svg
+      :align: center
+
+############
+TiddlyWiki 5
```

Comment:
Is there a prior version of tiddlywiki, that is still actively maintained? If not, please call the guide and the file just "tiddlywiki".